pkgs/by-name/de/degit-rs/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "degit-rs";
+  version = "0.1.3-unstable";
```

```suggestion
  version = "0.1.2-unstable-2021-09-22";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning